test/named_routes_test.rb

```diff
@@ -50,7 +50,7 @@
     end
 
     it 'recognizes glob string' do
-      @router.path(:glob).must_equal '/files/'
+      @router.path(:glob).must_equal '/files'
```

Comment:
This seemed like a insignificant change which i didn't it would effect anything else. But please if you know a reason why there should be that trailing slash there, let me know!